packages/components/src/Chip/Chip.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import styles from "./Chip.css";
+import { Avatar, AvatarProps } from "../Avatar";
+import { Text } from "../Text";
+import { ButtonDismiss } from "../ButtonDismiss";
+
+interface DismissibleAction {
```

Comment:
Hmmm maybe `DismissActionProp`? Consistent with how we name props. I.e., `ChipProps`

```suggestion
interface DismissActionProp {
```

Reply:
Interfaces which are used for React components should use the `Props` suffix, in this case `DimissibleAction` is a sub-interface so I don't think it needs to have the `Props` suffix.

Reply:
Ah that's fair!